sdk/src/cameras/ad-96tof1-ebz/camera_96tof1.cpp

```diff
@@ -430,6 +430,11 @@ aditof::Status Camera96Tof1::requestFrame(aditof::Frame *frame,
     uint16_t *frameDataLocation;
     frame->getData(FrameDataType::FULL_DATA, &frameDataLocation);
 
+    if (frameDataLocation == nullptr) {
+        LOG(WARNING) << "Failed to open frameDataLocation";
```

Comment:
- this should be an `ERROR`
- "Failed to open" is not really suggestive for this case as at this point we don`t know why the pointer is null. A more suggestive message would be "Received pointer frameDataLocation is null."

Reply:
In this case `frame` should be validated as it is the received argument, not frameDataLocation.